### PR TITLE
[Enhancement] Validate write single file parameter of unloading

### PR DESCRIPTION
**Pull request by letian-jiang**
Throw an exception if the parameter is not a boolean value.

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: fe/fe-core/src/main/java/com/starrocks/sql/ast/InsertStmt.java [hunk 1]
```diff
@@ -279,7 +279,13 @@ public Table makeTableFunctionTable() {
 
         // parse table function properties
         Map<String, String> props = getTableFunctionProperties();
-        boolean writeSingleFile = Boolean.parseBoolean(props.get("single"));
+        String single = props.get("single");
```
Comment (Smith-Cruise) [inline fe/fe-core/src/main/java/com/starrocks/sql/ast/InsertStmt.java:282]: .getOrDefault

**Review by Smith-Cruise — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]

:heart_eyes: **pass** : 4 / 4 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/ast/InsertStmt.java|4|4|100.00%|[]|

**Review by dirtysalt — APPROVED** [review]

**Review by Youngwb — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.2

**Pull request merged**